src/tests/reducers/authReducer.test.js

```diff
@@ -0,0 +1,37 @@
+import * as ActionType from "../../store/actions";
+import authReducer from "../../store/reducers/authReducer";
+
+test("Auth reducer returns default initialState", () => {
+  const action = { type: "dummy_action" };
+  const initialState = {
+    loginStatus: null,
+    isAuth: false,
+    userId: null,
+    jwt: null
+  };
+  expect(authReducer(undefined, action)).toEqual(initialState);
+});
+test("Auth reduer returns correct state after user logged out", () => {
+  const action = { type: ActionType.LOGGING_OUT };
+  const expectedState = {
+    loginStatus: null,
+    isAuth: false,
+    userId: null,
+    jwt: null
+  };
+
+  expect(authReducer(undefined, action)).toEqual(expectedState);
+});
+test("Auth reducer returns correct state after user is login", () => {
+  const action = {
+    type: ActionType.LOGIN,
+    payload: { userId: 1, jwt: "jwtToken", status: "login" }
+  };
+  const expectedState = {
+    isAuth: true,
+    userId: 1,
+    jwt: "jwtToken",
+    loginStatus: "login"
+  };
+  expect(authReducer(undefined, action)).toEqual(expectedState);
```

Comment:
its not very realistic that your current state is undefined when an action comes through.